doc/releases/nautilus.rst

```diff
@@ -216,13 +216,24 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
-#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
-   that ceph-volume can activate them after a host reboot.  On each
-   host containing OSDs,::
+#. If there are any OSDs in the cluster deployed with ceph-disk (e.g.,
```

Comment:
We need to emphasize that OSDs *need to be running*, otherwise a system-wide scan will not pick them up